packages/dapi-grpc/clients/core/v0/nodejs/CorePromiseClient.js

```diff
@@ -24,8 +24,10 @@ const {
       platform: {
         dapi: {
           v0: {
-            GetStatusRequest: PBJSGetStatusRequest,
-            GetStatusResponse: PBJSGetStatusResponse,
+            GetCoreChainStatusRequest: PBJSGetCoreChainStatusRequest,
```

Comment:
We already in Core service. There is no need to point to Core again. I think we agreed on `getBlockchainStatus` (similar to Core RPC command name).